activesupport/test/logger_test.rb

```diff
@@ -17,6 +17,14 @@ def setup
     @logger  = Logger.new(@output)
   end
 
+  def test_log_outputs_to
+    assert Logger.logger_outputs_to?(@logger, @output),          "Expected logger_outputs_to? @output to return true but was false"
+    assert Logger.logger_outputs_to?(@logger, @output, STDOUT),  "Expected logger_outputs_to? @output or STDOUT to return true but was false"
+
+    assert ! Logger.logger_outputs_to?(@logger, STDOUT),         "Expected log_outputs? to STDOUT to return false, but was true"
```

Comment:
Also better to not align the messages, if we need to change something in one line we would have to change in others too.

The message is also using `log_outputs?` instead of `logger_outputs_to?`